bittensor/_synapse/text_seq2seq_impl.py

```diff
@@ -147,13 +147,12 @@ def check_forward_request_tensor     ( self, forward_request_tensor ):
     def check_forward_response_tensor    ( self, forward_request_tensor, forward_response_tensor ):
         if forward_response_tensor == None:
             raise ValueError('Empty Response')
-
         if (
              len( forward_response_tensor.shape ) != 2 or
              forward_response_tensor.size(0) != forward_request_tensor.size(0) or
-             forward_response_tensor.size(1) != self.num_to_generate
+             forward_response_tensor.size(1) > self.num_to_generate
```

Comment:
what change necessitates the more lenient check here?

Reply:
The check was improperly formatted, num_to_generate only dictates the maximum length of the generation.